### DISCUSSION TITLE
RTL8812AU support for OVA config

### DISCUSSION THREAD

**Discussion by Haych** [home-assistant/operating-system#2275] [Ideas]
Hi, I am not even sure this is the correct place to start a discussion, but I was recently trying to get a USB Wifi dongle (Netgear A6100 - USB\VID_0846&PID_9052) working on my OVA based HAOS system. It doesnt work with the latest operating system version and I saw that support was added in some of the other configs - as per pull request 1669 - https://github.com/home-assistant/operating-system/pull/1669/commits/f20f8b9e3c0ccc44319a6fa8173ab1cd53c2314d

Is it possible to get the same change applied to ova_defconfig also?

Cheers

**Comment by agners** [comment]
Sure, this can be added easily. Created a PR: https://github.com/home-assistant/operating-system/pull/2279

**Reply by Haych** [reply]
Thanks Stefan, appreciate it.
